### DISCUSSION TITLE
Could not fetch data from API

### DISCUSSION THREAD

**Discussion by firefly07** [giejay/Immich-Android-TV#35] [Q&A]
Hello, 

cannot really get the app to work on my chromecast. After logging in via phone with Hostname+API, I get the follwoing error:
Could not fetch data from API, response: Failed to connect to /_my.ip_:80

Immich is working fine from my browser (http://_my.ip_:2283) and from my mobile app. I can also ping my chromecast from the VM that is hosting immich (running from proxmox). I am not very familiar with firewall configuration, so I did not touch anything there yet.

Any help or suggestion would be greatly appreciated

**Comment by giejay** [comment]
Did you include the port in the hostname?

**Reply by firefly07** [reply]
Thanks for following up, but I realized I made a mistake.
When I saw the error, I tried several times, also using other ports associated with immich (not the server port) and at some point I used the wrong port (swapped the last two digits), which gave me the error code above.

I should have instead posted the error code that I get when I use the correct port:
Could not fetch data from the API, response: Unable to parse TLS packet header

Sorry for posting the wrong error code. 

I recently update the immich server to latest version (6.0.X), but I still have the same error. Is the TV app version specific?
